crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -655,15 +695,41 @@ impl<M: NetworkMsg, K: SignatureKey + 'static> ConnectedNetwork<M, K> for Libp2p
             }
         };
 
+        // TODO maybe we should lift the metrics mutex up a level and copy the inner pattern
+        // ask during pair programming
+        // or maybe channels would be better?
```

Comment:
Same here, we can remove it or linked an issue.

Reply:
These comments are stale and should be deleted!